src/actions/favorites.js

```diff
@@ -0,0 +1,36 @@
+const FAVORITE_ADD = 'FAVORITE_ADD'
+const FAVORITE_REMOVE = 'FAVORITE_REMOVE'
+const FAVORITES_GET = 'FAVORITES_GET'
+const FAVORITES_FETCHED = 'FAVORITES_FETCHED'
+const FAVORITES_FAILED = 'FAVORITES_FAILED'
+const FAVORITE_SEARCH = 'FAVORITE_SEARCH'
+const FAVORITE_SEARCHED = 'FAVORITE_SEARCHED'
+
+const addFavorite = payload => ({ type: FAVORITE_ADD, payload})
```

Comment:
👍 , you can also create your own `createAction` helper, or use the one provided by `redux-action`.